### ISSUE TITLE
Difference in scoring methods of train and test

### ISSUE THREAD

**Issue by saatviga-healx** [thunlp/OpenKE#156]
Hi,

I see that the scoring calculation in the train method use tf.reduce_sum() while in the prediction function it is -tf.reduce_sum(). Can you explain why this is there? 

Thanks

**Comment by ShulinCao** [comment]
The distance is -tf.reduce_sum(), and the loss function is -log(sigmoid(-ground_truth * distance)), which can also be calculated as softplus(ground_truth*distance). You can refer to the codes in https://github.com/thunlp/OpenKE/blob/2380e9d724ab6cafbcc70fe2e511669802085db9/models/DistMult.py#L22 which is clear.
